src/qiskit_sphinx_theme/__init__.py

```diff
@@ -68,26 +68,13 @@ def remove_thebe_if_not_needed(
 
 
 def activate_themes(app: sphinx.application.Sphinx, config: sphinx.config.Config) -> None:
-    if config.html_theme == "qiskit":
-        # We set a low priority so that our Qiskit CSS file overrides Furo.
-        app.add_css_file("styles/furo.css", 100)
```

Comment:
We no longer need to add `furo.css` because it gets included in `qiskit-sphinx-theme.css` thanks to Webpack.